os/drivers/input/touchscreen.c

```diff
@@ -512,7 +515,14 @@ int touch_register(const char *path, struct touchscreen_s *dev)
 	}
 	touch_semgive(&dev->pollsem);
 #endif
-	int ret = register_driver(path, &g_touchdev_fileops, 0666, dev);
+	is_silent_mode = silent_reboot_is_silent_mode();
+	if (is_silent_mode) {
+		if (dev->ops && dev->ops->touch_disable) {
+			dev->ops->touch_disable(dev);
```

Comment:
When touch enable is called?
I think we should enable touch after lcd ON

Reply:
Application will turn on when silent mode